### DISCUSSION TITLE
Unable to click on commit graph to see a day's commits anymore

### DISCUSSION THREAD

**Discussion by kishore-ganesh** [community/community#115233] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Before, I used to be able to click on a date on my commit graph to see the commits I made on that day. This no longer works. This was a very useful feature - please see if you can restore it

**Comment by rdlf0** [comment]
Seems like they have recently changed some javascript around this, because the tooltip is now also pretty annoying. Previously it was following your cursor when you point different squares and now it is impossible to point the square above the one you are currently pointing. And yes, inability to click on squares is also really annoying.
